doc/modules/model_evaluation.rst

```diff
@@ -580,7 +580,9 @@ or *informedness*.
     * Balanced Accuracy as described in [Urbanowicz2015]_: the average of sensitivity and specificity
       is computed for each class and then averaged over total number of classes.
 
-.. topic:: References:
+|details-start|
+**References**
+|details-split|
```

Comment:
Sorry just a last comment, I just noticed some of the references (those with the format `[authorDATE]`) are called in some other parts of this page, but as of today, dropdowns break the sphynx cross-referencing system. Can you please revert the reference back to a `topic` directive whenever at least one of the references are called elsewhere?

Reply:
No problem! I've reinstated the topic directives when needed in the commit below. I had to remove a couple of dropdowns but most remained